packages/rx_bloc_cli/mason_templates/bricks/feature_otp/__brick__/lib/feature_otp/views/otp_screen.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/text_field_dialog.dart';
+import 'package:widget_toolkit/theme_data.dart';
+import 'package:widget_toolkit_otp/widget_toolkit_otp.dart';
+
+import '../../app_extensions.dart';
+import '../fake_classes/custom_sms_code_service.dart';
+import '../fake_classes/otp_text_field_validator.dart';
+
+class OtpScreen extends StatelessWidget {
+  const OtpScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final String phoneNumber = context.l10n.featureOtp.phoneNumber;
```

Comment:
Why are you creating this variable `phoneNumber`  here instead of not just using `context.l10n.featureOtp.phoneNumber` it at line 45 like: `label:context.l10n.featureOtp.phoneNumber`

Reply:
If I use it like that, it will give me an error, weird behaviour but yea